internal/cmd/bundle/bundle.go

```
@@ -134,7 +134,12 @@ https://github.com/CodeIntelligenceTesting/cifuzz/issues`, opts.BuildSystem, sys
 				return cmdutils.WrapSilentError(err)
 			}
 
-			opts.FuzzTests = args
+			fuzzTests, err := cmdutils.FuzzTestArgument(opts.ResolveSourceFilePath, args, opts.BuildSystem, opts.ProjectDir)
+			if err != nil {
+				return cmdutils.WrapSilentError(err)
```

This silences the error without printing it. Print it first via `log.Error(err)`  to avoid that.